index.js

```diff
@@ -3,35 +3,43 @@
 // console.log("%cQuestion 1", "color: red") 
 
 // console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+// => 2
+console.log("%c----------", "color: red") 
 
+console.log("%cQuestion 1", "color: red") 
+counter = 1
+counter = 2
+console.log(counter)
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+const name = "Kwaku"
 
 // console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+ console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
 
+console.log("%cQuestion 3", "color: red") 
 
+function drinkWater(thirst) {
+  console.log("Man I sure am thristy")
+  thirst -= 1
+  console.log("Ahh that hits the spot")
+  return thirst
+}
```

Comment:
this is great! on line 37 you could also do thirst--
you could also combine lines 37 and 39 and just do
return thirst - 1